xform-marc21-to-xml/src/main/java/edu/stanford/AuthDBProperties.java

```diff
@@ -76,6 +80,37 @@ private void initDataSourceProperties(Properties properties) {
         this.userPass = properties.getProperty("PASS");
     }
 
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
Maybe this code could follow advice in that last SO answer, i.e. "Use the excellent helper classes EqualsBuilder and HashCodeBuilder from the Apache Commons Lang library."